### ISSUE TITLE
No encuentra Procfile

### ISSUE THREAD

**Issue by Dunkan1** [IIC2233/Syllabus#711]
cuando hago push de mi aplicacion al parecer no esta subiendo el procfile lo que produce que no se ejecute en el servidor

![capture](https://user-images.githubusercontent.com/26102365/27513292-ca979cfe-592e-11e7-906d-e660844f4506.PNG)

**Comment by MainScientist** [comment]
No veo por qué está fallando. Estás seguro que el archivo no tiene ninguna extensión y que se encuentra agregado y commiteado al repositorio?

**Comment by Dunkan1** [comment]
el archivo no tiene ninguna extencion, lo borre, lo volvi a crear y nada, lo que sucede es que al hacer git add . no agrega el Procfile y por lo tanto no se sube al repositorio.

en este link
https://stackoverflow.com/questions/36300215/trying-to-deploy-to-heroku-but-keep-getting-rejected
encontre esto, que es un error similar al mio
"""
Coming to this rather late, so not sure if this will be helpful. I found your question because I was having the same problem: I used the Heroku-supplied Django template and definitely had a Procfile in the root of the project.

Which folder did you do $ git init in?

My mistake was that I had created a folder ('WEB_CODE') on my desktop to hold this project and initialized THIS folder as a git repo. This meant that the Django app was being installed in a sub-folder at WEB_CODE/myapp, so the Procfile was at the root of myapp, which was not the head of the git repo.

I scrapped this and started over. I now have a new folder to hold everything (suppose it's called 'APP_CODE'), and once I installed the Heroku-supplied Django app, I made the subdirectory which was created from this installation the head of the git repo (eg: APP_CODE/myapp is the repo, not APP_CODE). Consequently, the Procfile was then in the root of the project.

I committed the changes, pushed to my personal github, and then I re-tried the $heroku create and $ git push heroku master commands and they worked like a charm.
"""
pero no se como realizar lo que se indica
